### PR TITLE
feat(table): support close fn to close filter dropdown only

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

- feat: #37665 

### 💡 Background and solution

Provide a method `close` to close dropdown will not confirm.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    support close fn to close filter dropdown only       |
| 🇨🇳 Chinese |    提供一个`close`方法用以仅关闭筛选弹出框       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: components/table/interface.tsx [hunk 1]
```diff
@@ -83,10 +83,16 @@ export interface FilterDropdownProps {
...
+  /**
+   * Confirm filter value, if you want to close dropdown before commit, you can call with
+   * {closeDropdown: true}
+   */
   confirm: (param?: FilterConfirmProps) => void;
   clearFilters?: () => void;
   filters?: ColumnFilterItem[];
   visible: boolean;
+  /** Only close filterDropdown */
+  close: () => void;
```
Comment (afc163) [inline components/table/interface.tsx:95]: Is `visible` a better solution to control visibililty of filterDropdown?

Comment (kiner-tang) [reply components/table/interface.tsx:95]: if use `visible` to control visibility, we should use a state to manage this state outside the filterDropdown。But this state usually does nothing else。So I think using the 'close' method might be a better choice

**Comment by afc163** [comment]
feature branch please

**Comment by kiner-tang** [comment]
> feature branch please

ok

**Comment by kiner-tang** [comment]
> feature branch please

- https://github.com/ant-design/ant-design/pull/37745

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/37745

**Pull request closed without merging**